coinlib/src/main/java/com/keystone/coinlib/utils/AbiLoadManager.java

```diff
@@ -0,0 +1,154 @@
+package com.keystone.coinlib.utils;
+
+import android.database.Cursor;
+import android.database.SQLException;
+import android.database.sqlite.SQLiteDatabase;
+import android.text.TextUtils;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.File;
+
+import static com.keystone.coinlib.v8.ScriptLoader.readAsset;
+
+public class AbiLoadManager {
+    private static final String DATABASE_TFCARD_PATH = "contracts" + File.separator + "ethereum";
+    private static final String SELF_DEFINE_TFCARD_PATH = "contracts" + File.separator + "self_define";
+    private String address;
+    private boolean isFromTFCard;
+    private Contract contract;
+    private JSONObject bundleMap;
+
+    public AbiLoadManager(String address) {
+        this.address = address.toLowerCase();
+        contract = new Contract();
+        try {
+            bundleMap = new JSONObject(readAsset("abi/abiMap.json"));
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Contract loadAbi() {
+        if (TextUtils.isEmpty(address)) {
+            return contract;
+        }
+        loadFromAssets();
```

Comment:
Create an ABIStoreEngine interface like 
```
public interface IABIStoreEngine {
    public Result/*or anything*/ load()
}
```
Then create 3 new class implement this interface and create instances in ABILoadManager's constructor would be better.
They would be InternalABIStore, TFCardABIStore, SelfDefinedABIStore